conbench/entities/benchmark_result.py

```diff
@@ -987,24 +987,27 @@ class BenchmarkResultStatsSchema(marshmallow.Schema):
         },
     )
     times = marshmallow.fields.List(
+        # https://github.com/conbench/conbench/issues/1399
         marshmallow.fields.Decimal(allow_none=True),
-        required=True,
+        required=False,
         metadata={
             "description": conbench.util.dedent_rejoin(
                 """
-                A list of benchmark durations. If `data` is a duration measure,
-                this should be a duplicate of that object. The values should be
-                ordered in the order the iterations were executed (the first
-                element is the first iteration, the second element is the
-                second iteration, etc.). If an iteration did not complete but
-                others did and you want to send partial data, mark each
-                iteration that didn't complete as `null`.
-
-                You may populate both this field and the "error" field in the top level
-                of the benchmark result payload. In that case, this field measures
-                how long the benchmark took to run before the error occurred. These
-                values will not be compared to non-errored values in analyses and
-                comparisons.
+                Here, you can provide a list of benchmark durations. That can
+                make sense if `data` is not a duration measure.
+
+                Optional. If provided, must be a list of numbers. `null` is
+                allowed to represent a failed repetition.
```

Comment:
Clarified the intent how we see it as of today.

Used the newer 'repetition' language here. I forgot that I started doing so a longer while ago!